scripts/download_cities.py

```diff
@@ -0,0 +1,37 @@
+from pathlib import Path
+
+import pandas as pd
+import requests
+import typer
+
+
+def download_cities(
+    output_path: Path = typer.Argument(
+        "edsnlp/resources/cities.csv.gz",
+        help="Path to the output CSV table.",
+    ),
+) -> None:
+    """
+    Convenience script to automatically download a list of French cities.
+    """
+
+    typer.echo("Downloading data...")
+
+    r = requests.get(
+        "https://www.data.gouv.fr/fr/datasets/r/34d4364c-22eb-4ac0-b179-7a1845ac033a"
+    )
```

Comment:
Should we move the URL to the script arguments ?

```suggestion
def download_cities(
    output_path: Path = typer.Argument(
        "edsnlp/resources/cities.csv.gz",
        help="Path to the output CSV table.",
    ),
    url : Path = typer.Argument(
        "https://www.data.gouv.fr/fr/datasets/r/34d4364c-22eb-4ac0-b179-7a1845ac033a",
        help="URL to the cities JSON file",
) -> None:
    """
    Convenience script to automatically download a list of French cities.
    """

    typer.echo("Downloading data...")

    r = requests.get(url)
```